UM/Settings/ContainerRegistry.py

```diff
@@ -166,36 +167,58 @@ def addContainerType(cls, container):
     #   If this function is called again, it will clear the old data and reload.
```

Comment:
Fixed the comment and made sure we actually do not re-add already existing containers. (Since that was rather bad behaviour.)